collect_app/src/main/java/org/odk/collect/android/widgets/ImageWidget.java

```diff
@@ -150,11 +148,7 @@ private void captureImage() {
         errorTextView.setVisibility(View.GONE);
         Intent intent;
         if (selfie) {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-                intent = new Intent(getContext(), CaptureSelfieActivityNewApi.class);
-            } else {
-                intent = new Intent(getContext(), CaptureSelfieActivity.class);
```

Comment:
Looks like `CaptureSelfieActivity` should be removed at all because it's unused.